### PR TITLE
memory speed units and plural for memory units

### PR DESCRIPTION
**Pull request by beradrian**

### CODE REVIEW

**Comment by beradrian** [comment]
Fix for issue #42 and #43.

**Comment by gentooboontoo** [comment]
Thank you. Could you split your changes in two commits (memory plurals then memory speeds)?

File: src/quantities.js [hunk 1]
```diff
@@ -235,8 +235,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
     "<rpm>"   :[["rpm"], 2.0*Math.PI / 60.0, "angular_velocity", ["<radian>"], ["<second>"]],
 
     /* memory */
-    "<byte>"  :[["B","byte"], 1.0, "memory", ["<byte>"]],
-    "<bit>"  :[["b","bit"], 0.125, "memory", ["<byte>"]],
+    "<byte>"  :[["B","byte","bytes"], 1.0, "memory", ["<byte>"]],
+    "<bit>"  :[["b","bit","bits"], 0.125, "memory", ["<byte>"]],
+
+    /* memory speed */
+    "<Bps>" : [["Bps"], 1.0, "speed", ["<byte>"], ["<second>"]],
```
Comment (gentooboontoo) [inline src/quantities.js]: `Bps` is not a kind of `speed` (It cannot be converted to `m/h` for instance). `speed` is length over time.

Comment (beradrian) [inline src/quantities.js]: Then the kind should be `memory speed`.

**Comment by beradrian** [comment]
I split the changes into different commits as you requested.

**Comment by beradrian** [comment]
How should I add `memory_speed` to `SIGNATURE_VECTOR` and `KINDS`?

**Comment by beradrian** [comment]
Here's an example http://plnkr.co/edit/mHjLqo0p5mhv8KPdI785?p=preview

File: src/quantities.js [hunk 2]
```diff
@@ -235,8 +235,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
     "<rpm>"   :[["rpm"], 2.0*Math.PI / 60.0, "angular_velocity", ["<radian>"], ["<second>"]],
 
     /* memory */
```
Comment (gentooboontoo) [inline src/quantities.js]: Could you replace `memory` by `information` everywhere in the file to keep the library on par with _Ruby-units_?

File: src/quantities.js [hunk 3]
```diff
@@ -235,8 +235,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
...
-    "<byte>"  :[["B","byte"], 1.0, "memory", ["<byte>"]],
-    "<bit>"  :[["b","bit"], 0.125, "memory", ["<byte>"]],
+    "<byte>"  :[["B","byte","bytes"], 1.0, "memory", ["<byte>"]],
+    "<bit>"  :[["b","bit","bits"], 0.125, "memory", ["<byte>"]],
+    
+    /* memory speed */
+    "<Bps>" : [["Bps"], 1.0, "memory_speed", ["<byte>"], ["<second>"]],
```
Comment (gentooboontoo) [inline src/quantities.js]: Please use `information-rate` instead of `memory_speed`

**Comment by beradrian** [comment]
I renamed memory to information and memory_speed to information-rate. What about my question regarding SIGNATURE_VECTOR and KINDS?

**Comment by gentooboontoo** [comment]
You don't have to change `SIGNATURE_VECTOR`. Just add `"25599999980": "information_rate"` to `KINDS` (definitively use underscore for kinds, not dash, sorry for the inconvenience).

Please also include tests about bps/Bps and kind.

File: src/quantities.js [hunk 4]
```diff
@@ -234,9 +234,13 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
...
 
-    /* memory */
-    "<byte>"  :[["B","byte"], 1.0, "memory", ["<byte>"]],
-    "<bit>"  :[["b","bit"], 0.125, "memory", ["<byte>"]],
+    /* information */
+    "<byte>"  :[["B","byte","bytes"], 1.0, "information", ["<byte>"]],
+    "<bit>"  :[["b","bit","bits"], 0.125, "information", ["<byte>"]],
+    
+    /* information rate */
+    "<Bps>" : [["Bps"], 1.0, "information-rate", ["<byte>"], ["<second>"]],
```
Comment (gentooboontoo) [inline src/quantities.js]: dash to be replaced with underscore.

**Comment by beradrian** [comment]
I added "information_rate" to KINDS and replaced dash with underscore.
I saw no tests for memory/information. Are you sure you want me to add for information_rate?

**Comment by gentooboontoo** [comment]
Merged. I have rebased your commits and refactored tests. Thank you.

**Pull request closed without merging**